File: backend/app/modelos/Trabajos_aceptados.go
package modelos

type Trabajo_aceptado struct {
	Id_trabajo int `json:"id_trabajo"`
	Id_usuario int `json:"id_usuario"`
	Fecha_aceptacion string `json:"fecha_aceptacion"`
	Estado_trabajo string `json:"estado_trabajo"`
}

type ListaTrabajosAceptados struct{
	ListaTrabajos []Trabajo_aceptado `json:"lista_trabajos"`
}

File: backend_Python/app/modelos/Trabajos_Aceptados.py
class TrabajoAceptado:
    def __init__(self, id_trabajo, id_usuario, fecha_aceptacion, estado_trabajo):
        self.id_trabajo = id_trabajo
        self.id_usuario = id_usuario
        self.fecha_aceptacion = fecha_aceptacion
        self.estado_trabajo = estado_trabajo

    def to_dict(self):
        return {
            "id_trabajo": self.id_trabajo,
            "id_usuario": self.id_usuario,
            "fecha_aceptacion": self.fecha_aceptacion,
            "estado_trabajo": self.estado_trabajo
        }